package.json

```diff
@@ -14,9 +14,12 @@
   },
   "dependencies": {
     "bootstrap": "^3.3.5",
+    "exports-loader": "^0.6.2",
+    "imports-loader": "^0.6.4",
```

Comment:
Both of webpack loaders are dev time libraries, you should put it in devDependencies